src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventPipeEventProvider.cs

```diff
@@ -73,9 +73,9 @@ uint IEventProvider.EventUnregister(long registrationHandle)
         }
 
         // Get or set the per-thread activity ID.
-        int IEventProvider.EventActivityIdControl(Interop.Advapi32.ActivityControl ControlCode, ref Guid ActivityId)
+        int IEventProvider.EventActivityIdControl(Interop.Advapi32.ActivityControl controlCode, ref Guid activityId)
```

Comment:
These come from
https://github.com/dotnet/runtime/blob/7b2925b9d3920957f35a58f073e36556b18056be/src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/IEventProvider.cs#L38
We should be fine to rename that as well